requirements.txt

```diff
@@ -1,2 +1,2 @@
-lxml==4.5.1
+lxml==4.6.2
```

Comment:
Bumped to address an `lxml` vulnerability as identified by Snyk. I have retested to ensure that this doesn't impact the CPEs that are generated or the values added to the XML.

![image](https://user-images.githubusercontent.com/20306699/101211889-7e9c2780-363d-11eb-9a98-f0e6a22d4248.png)